datahub-actions/setup.py

```diff
@@ -146,6 +146,7 @@ def get_long_description():
         "executor = datahub_actions.plugin.action.execution.executor_action:ExecutorAction",
         "slack = datahub_actions.plugin.action.slack.slack:SlackNotificationAction",
         "teams = datahub_actions.plugin.action.teams.teams:TeamsNotificationAction",
+        "metadata_change_sync = datahub_actions.plugin.action.metadata_change_sync.metadata_change_sync:MetadataChangeSyncAction",
```

Comment:
@hsheth2 hope this is the right place to put for dynamic discovery.

Reply:
yep looks good!